teamsyncd/teamsync.cpp

```diff
@@ -65,7 +66,7 @@ void TeamSync::addLag(const string &lagName, int ifindex, bool admin_state,
         return;
 
     /* Start track the team instance */
-    TeamPortSync *sync = new TeamPortSync(lagName, ifindex, &m_lagTable);
+    TeamPortSync *sync = new TeamPortSync(lagName, ifindex, &m_lagMemberTable);
     m_select->addSelectable(sync);
     m_teamPorts[lagName] = shared_ptr<TeamPortSync>(sync);
```

Comment:
>shared_ptr [](start = 27, length = 10)

If the raw pointer is going to be inside a shared_ptr, why not use make_shared() at the beginning, and make it more exception-safe.